src/pages/ReimbursementAccount/NonUSD/SignerInfo/index.tsx

```diff
@@ -26,54 +29,70 @@ type SignerInfoProps = {
     onSubmit: () => void;
 };
 
-type SignerDetailsFormProps = SubStepProps & {isSecondSigner: boolean};
+type SignerDetailsFormProps = SubStepProps;
 
 // eslint-disable-next-line @typescript-eslint/no-unsafe-member-access, @typescript-eslint/no-unsafe-assignment
 const SUBSTEP: Record<string, number> = CONST.NON_USD_BANK_ACCOUNT.SIGNER_INFO_STEP.SUBSTEP;
 const {OWNS_MORE_THAN_25_PERCENT, COMPANY_NAME} = INPUT_IDS.ADDITIONAL_DATA.CORPAY;
 
-const bodyContent: Array<ComponentType<SignerDetailsFormProps>> = [Name, JobTitle, DateOfBirth, UploadDocuments, Confirmation];
+const fullBodyContent: Array<ComponentType<SignerDetailsFormProps>> = [Name, JobTitle, DateOfBirth, Address, UploadDocuments, Confirmation];
 const userIsOwnerBodyContent: Array<ComponentType<SignerDetailsFormProps>> = [JobTitle, UploadDocuments, Confirmation];
 
 function SignerInfo({onBackButtonPress, onSubmit}: SignerInfoProps) {
     const {translate} = useLocalize();
 
+    const [directorKeys, setDirectorKeys] = useState<string[]>([]);
     const [reimbursementAccount] = useOnyx(ONYXKEYS.REIMBURSEMENT_ACCOUNT);
     const [reimbursementAccountDraft] = useOnyx(ONYXKEYS.FORMS.REIMBURSEMENT_ACCOUNT_FORM_DRAFT);
+    const [account] = useOnyx(ONYXKEYS.ACCOUNT);
     const policyID = reimbursementAccount?.achData?.policyID;
     const [policy] = useOnyx(`${ONYXKEYS.COLLECTION.POLICY}${policyID}`);
     const currency = policy?.outputCurrency ?? '';
-    // TODO set this based on param from redirect or BE response
-    const isSecondSigner = false;
     const isUserOwner = reimbursementAccount?.achData?.corpay?.[OWNS_MORE_THAN_25_PERCENT] ?? reimbursementAccountDraft?.[OWNS_MORE_THAN_25_PERCENT] ?? false;
     const companyName = reimbursementAccount?.achData?.corpay?.[COMPANY_NAME] ?? reimbursementAccountDraft?.[COMPANY_NAME] ?? '';
-
+    const bankAccountID = reimbursementAccount?.achData?.bankAccountID ?? CONST.DEFAULT_NUMBER_ID;
     const [currentSubStep, setCurrentSubStep] = useState<number>(SUBSTEP.IS_DIRECTOR);
     const [isUserDirector, setIsUserDirector] = useState(false);
 
-    const submit = () => {
-        if (currency === CONST.CURRENCY.AUD) {
-            setCurrentSubStep(SUBSTEP.ENTER_EMAIL);
-        } else {
-            onSubmit();
-        }
-    };
+    const submit = useCallback(() => {
+        const {signerDetails, signerFiles} = getSignerDetailsAndSignerFilesForSignerInfo(reimbursementAccountDraft, account?.primaryLogin ?? '', isUserOwner);
 
-    const handleNextSubStep = (value: boolean) => {
-        if (currentSubStep !== SUBSTEP.IS_DIRECTOR) {
+        saveCorpayOnboardingDirectorInformation({
+            inputs: JSON.stringify(signerDetails),
+            ...signerFiles,
+            bankAccountID,
+            directorIDs: `${directorKeys.toString()}`,
```

Comment:
We don't need this anymore since we're doing them one at a time

Reply:
alright, I'll remove it, thanks

Reply:
resolved